app/templates/admin/permissions/event-roles.hbs

```diff
@@ -17,18 +17,19 @@
     </tr>
   </thead>
   <tbody>
-    {{#each model.roles as |item|}}
+    {{#each model.roles as |role|}}
       <tr>
-        <td>{{item}}</td>
-        {{#each (range 1 6) as |number|}}
+        <td>{{role}}</td>
+        {{#each (range 1 6) as |index|}}
```

Comment:
@CosmicCoder96 @niranjan94 I think it would me more appropriate if we close this PR, and do this in a separate issue. What do you think?